File: server/src/main/java/com/sju18/petmanagement/domain/pet/pet/dao/Pet.java
package com.sju18.petmanagement.domain.pet.pet.dao;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDate;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Pet {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "pet_id")
    private Long id;

    @Column(nullable = false)
    private String ownername;
    @Column(nullable = false)
    private String name;

    @Column
    private String species;
    private String breed;
    private LocalDate birth;
    private Boolean yearOnly;
    private Boolean gender;
    private String message;
    private String photoUrl;
}

File: client/android/app/src/main/java/com/sju18001/petmanagement/restapi/dao/Pet.kt
package com.sju18001.petmanagement.restapi.dao

data class Pet(
    val id: Long,
    val ownername: String,
    val name: String,
    val species: String,
    val breed: String,
    val birth: String?,
    val yearOnly: Boolean?,
    val gender: Boolean,
    val message: String?,
    val photoUrl: String?
)
